docker/docker-compose.yml

```diff
@@ -0,0 +1,147 @@
+version: "3.9"
+
+services:
+
+  setup-volumes:
+    # We can use any image we want as long as we can chown
+    image: busybox
+    # Need a user priviliged enough to chown
+    user: "root"
+    # Specify the group in question
+    volumes:
+      # The volumes to chown
+      - squad-frontend:/app/
+      - squad-storage:/storage/
+    command:
+      - /bin/sh
+      - -c
+      - |
+        chown -R 1000:1000 /app
+        chown -R 1000:1000 /storage
+
+  #
+  # QA-Reports frontend
+  #
+  squad-frontend:
+    image: squadproject/squad
+    command:
+      - /bin/sh
+      - -c
+      - |
+        sleep 10
+        squad --bind 0.0.0.0:8000 --timeout=60 --workers=8 --log-level DEBUG
+    environment:
+      DATABASE: "ENGINE=django.db.backends.postgresql_psycopg2:NAME=squad:HOST=postgres:USER=squad:PASSWORD=squad"
+      SQUAD_STATIC_DIR: "/app/static"
+      SQUAD_EXTRA_SETTINGS: "/app/extra_settings.py"
+      SQUAD_CELERY_BROKER_URL: "amqp://rabbitmq"
+      SQUAD_BASE_URL: "https://www.your-squad-url.org"
+      # SENTRY_DSN: "<sentry-dsn>"
+      SECRET_KEY_FILE: "/app/secret.dat"
+      SQUAD_SITE_NAME: "CIP SQUAD Site"
```

Comment:
Could you please leave it a generic name, just "SQUAD Site"?

Reply:
I missed this. Thanks for noting.

Changed with commit 149a5fa